Open-Source/Contribute/Readme.md

```diff
@@ -19,6 +20,7 @@
 - [DBpedia](https://github.com/dbpedia/)
 - [discord-bot](https://github.com/BeccaLyria/discord-bot) 
 - [EddieBot](https://github.com/EddieHubCommunity/EddieBot)
+- [Fossology](https://summerofcode.withgoogle.com/programs/2022/organizations/fossology)
```

Comment:
Please add name in alphabetical order

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient